sdks/java/io/amazon-web-services2/src/test/java/org/apache/beam/sdk/io/aws2/kinesis/KinesisReaderCheckpointTest.java

```diff
@@ -35,12 +35,10 @@
 import software.amazon.awssdk.services.kinesis.model.ShardIteratorType;
 
 public class KinesisReaderCheckpointTest {
-  /**
-   * This was generated with Beam master branch.
-   * https://github.com/apache/beam/commit/ca0787642a6b3804a742326147281c99ae8d08d2
-   */
+
+  /** This was generated manually. */
```

Comment:
nit Can you comment on how this was generated manually? Or you just used the failure to set as a baseline?

Reply:
Yeah, just used the error message which prints `serializedCheckpoint` to set the base line:

https://github.com/apache/beam/blob/c29485f28b34e2a5b162f5966b9a5b2d1948580b/sdks/java/io/amazon-web-services2/src/test/java/org/apache/beam/sdk/io/aws2/kinesis/KinesisReaderCheckpointTest.java#L91